deployer/deployer.yaml

```diff
@@ -88,6 +88,17 @@ items:
     subjects:
     - kind: ServiceAccount
       name: logging-deployer
+  -
+    apiVersion: v1
+    kind: RoleBinding
+    metadata:
+      name: logging-elasticsearch-view-role
+    roleRef:
+      kind: ClusterRole
+      name: view
+    subjects:
+    - kind: ServiceAccount
+      name: aggregated-logging-elasticsearch
```

Comment:
hmm, this may be vestigial from when I was trying to get the openshift-client to work within the openshift-elasticsearch-plugin.  let me verify whether or not this is needed still.